### PR TITLE
Various changes to organizations endpoint

### PR DESCRIPTION
**Pull request by hacosta**

### CODE REVIEW

File: moto/organizations/responses.py [hunk 1]
```diff
@@ -115,6 +120,11 @@ def list_policies(self):
             self.organizations_backend.list_policies(**self.request_params)
         )
 
+    def delete_policy(self):
+        return json.dumps(
```
Comment (bblommers) [inline moto/organizations/responses.py:124]: The backend-method doesn't seem to return anything. It probably reads better to rewrite this to:
```
self.organizations_backend.delete_policy(**self.request_params)
return json.dumps({})
```

File: tests/test_organizations/test_organizations_boto3.py [hunk 2]
```diff
@@ -410,28 +410,72 @@ def test_describe_policy_exception():
 
 
 @mock_organizations
-def test_attach_policy():
```
Comment (bblommers) [inline tests/test_organizations/test_organizations_boto3.py:413]: Don't think this test needs to change? The attach_policy functionality hasn't changed

File: tests/test_organizations/test_organizations_boto3.py [hunk 3]
```diff
@@ -479,6 +523,57 @@ def test_attach_policy_exception():
...
+    with assert_raises(ClientError) as e:
+        response = client.update_policy(PolicyId=non_existent_policy_id)
+    ex = e.exception
+    ex.operation_name.should.equal("DeletePolicy")
+    ex.response["Error"]["Code"].should.equal("400")
+    ex.response["Error"]["Message"].should.contain("PolicyNotFoundException")
+
+
+@mock_organizations
+def test_update_policy_exception():
```
Comment (bblommers) [inline tests/test_organizations/test_organizations_boto3.py:565]: Please use unique names for the tests - `test_update_nonexisting_policy_with_description`?

**Comment by hacosta** [comment]
Thanks for the feedback and sorry about the noise. LMK if this works.

**Comment by bblommers** [comment]
This is now part of moto >= 1.3.15.dev999

**Pull request merged**